compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STNodeTransformer.java

```diff
@@ -429,6 +429,10 @@ public T transform(STXMLComment xMLComment) {
         return transformSyntaxNode(xMLComment);
     }
 
+    public T transform(STXMLCDATANode xMLCDATANode) {
```

Comment:
This is consistent with the previous parameters. But it looks weird. XML is an abbreviation, not a single word. should be xmlCDATANode.

Reply:
Are we talking about the class name or the variable name?
Since both XML and CDATA are abbreviations, shouldn't the variable name be xmlCdataNode? Anyway, this is an auto-generated class by Treegen can't do anything unless we change the class name. 

I think for the class names, XmlCdataNode and STXmlCdataNode is the correct way. But since all other xml nodes have used XML, XMLCDATANode is fine?

Reply:
I think it's cleaner to rename all the `XML` nodes to the camel case. But will it cause backward compatibility issues?

Reply:
Let's go ahead with current approach. We need to review all these cases in current APIs. Could you please create an issue for this.

Reply:
Sure ayye

Reply:
Here is the issue #32671